addons/account/views/account_move_views.xml

```diff
@@ -999,6 +999,8 @@
                                                context="{'default_partner_id': commercial_partner_id}"
                                                domain="[('partner_id', '=', bank_partner_id)]"
                                                attrs="{'readonly': [('state', '=', 'paid')]}"/>
+                                        <field name="display_qr_code" invisible="1"/>
```

Comment:
I would like to move this field in the dedicated `!-- Invisible fields -->` section

Reply:
https://kaamelott-soundboard.2ec0b4.fr/#son/enfin-quand-cest-demande-gentiment